fakeredis.py

```diff
@@ -25,23 +26,24 @@
 
 PY2 = sys.version_info[0] == 2
 
-if not PY2:
-    long = int
-
 
 __version__ = '0.8.2'
 
 
-if sys.version_info[0] == 2:
+if PY2:
+    DEFAULT_ENCODING = 'utf-8'
```

Comment:
Only defining `DEFAULT_ENCODING` for py2 means you have to be really careful about never using `DEFAULT_ENCODING` in py3.  The intent here is to have all the constants defined for py2 as well as py3.